### PR TITLE
v1 nlp notebooks changed to reflect 1.46.0 SDK

### PR DESCRIPTION
**Pull request by DiegoRaygozaCastanos**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- all automl-nlp notebooks print outs changed to reflect current SDK, i.e. "This notebook was created using version 1.46.0 of the Azure ML SDK"
- automl-nlp-ner changed to have offline inference evaluation with seqeval metrics
- automl-nlp-multilabel imports changed to reflect refactoring, i.e. load_model_wrapper is imported from azureml.automl.dnn.nlp.common.io.utils now

fixes #

### CODE REVIEW

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multiclass/automl-nlp-text-classification-multiclass.ipynb [hunk 1]
```diff
@@ -104,7 +104,7 @@
    },
    "outputs": [],
    "source": [
-    "print(\"This notebook was created using version 1.39.0 of the Azure ML SDK\")\n",
+    "print(\"This notebook was created using version 1.46.0 of the Azure ML SDK\")\n",
```
Comment (arjsingh) [inline v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multiclass/automl-nlp-text-classification-multiclass.ipynb:107]: Should have shared this sooner, but last week when Miseon and I were discussing, we realized that it's better to remove this line else we'd need to update regularly.

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multilabel/automl-nlp-text-classification-multilabel.ipynb [hunk 2]
```diff
@@ -100,7 +100,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\"This notebook was created using version 1.39.0 of the Azure ML SDK\")\n",
+    "print(\"This notebook was created using version 1.46.0 of the Azure ML SDK\")\n",
```
Comment (arjsingh) [inline v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multilabel/automl-nlp-text-classification-multilabel.ipynb:103]: Better to remove this line/cell

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb [hunk 3]
```diff
@@ -95,7 +98,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\"This notebook was created using version 1.39.0 of the Azure ML SDK\")\n",
+    "print(\"This notebook was created using version 1.46.0 of the Azure ML SDK\")\n",
```
Comment (arjsingh) [inline v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb:101]: Better to remove this line/cell

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb [hunk 4]
```diff
@@ -439,8 +451,82 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pd.DataFrame(\n",
-    "    {\"metric name\": list(test_metrics.keys()), \"value\": list(test_metrics.values())}\n",
+    "def named_entity_recognition_report(predictions_file, test_data_file, use_str = True):\n",
```
Comment (arjsingh) [inline v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb:454]: Possible to move this to a utils.py kind of file in this folder? @msp8955 wdyt?

Comment (msp8955) [reply v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb:453]: yeah similar to forecasting example having metrics_helper.py
https://github.com/Azure/azureml-examples/tree/main/v1/python-sdk/tutorials/automl-with-azureml/forecasting-bike-share

**Review by msp8955 — APPROVED** [review]

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multilabel/automl-nlp-text-classification-multilabel.ipynb (hunk 2 above)
Comment (DiegoRaygozaCastanos) [reply v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multilabel/automl-nlp-text-classification-multilabel.ipynb]: Line removed

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb (hunk 3 above)
Comment (DiegoRaygozaCastanos) [reply v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb]: Line removed

File: v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb (hunk 4 above)
Comment (DiegoRaygozaCastanos) [reply v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb]: Done

**Review by arjsingh — APPROVED** [review]

**Review by jeff-shepherd — APPROVED** [review]

**Pull request merged**